mypy/expandtype.py

```diff
@@ -94,6 +94,10 @@ def visit_union_type(self, t: UnionType) -> Type:
     def visit_partial_type(self, t: PartialType) -> Type:
         return t
 
+    def visit_type_type(self, t: TypeType) -> Type:
+        item = t.item.accept(self)
```

Comment:
I suspect that we should verify here that the new item type is valid (instance or union of instances, or `Any`). However, `expand_type` currently can't report an error, which is unfortunate. You could just write a TODO comment about this and not worry about it for now.